### ISSUE TITLE
If MultiMC directory contains Cyrillic symbols, shortcuts created by the launcher will be invalid

### ISSUE THREAD

**Issue by TheDanikReal** [MultiMC/Launcher#5009]
### Operating System

Windows

### Description of bug

If directory contains Cyrillic symbols (for example: C://программы/multimc) shortcuts created by launcher will be invalid 

### Steps to reproduce

* Put MultiMC in folder which contains Cyrillic symbols
* Open launcher
* Click on any minecraft version and click Create shortcut

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Feniksovich** [comment]
You should not use Cyrillic symbols in paths. Never and anywhere. Really.

**Comment by gvmii** [comment]
what is that response? lmao? that's the windows path when you set your name, and guess what, some people might write their name in Cyrillic...

**Comment by TheDanikReal** [comment]
Yes, I keep all the programs in my folder and call them Cyrillic names, since it's my native language. But now I use English names for folders

**Comment by Feniksovich** [comment]
> what is that response? lmao? that's the windows path when you set your name, and guess what, some people might write their name in Cyrillic...

My answer should be taken as advice. Meaning Cyrillic characters in fact I meant any characters in general other than Latin and special characters (allowed to use in file names in a your file system). 
This problem is common, so the best solution in my opinion is to stop using such characters in paths.

Best regards, Feniksovich from Russia :)

**Comment by TheDanikReal** [comment]
> > what is that response? lmao? that's the windows path when you set your name, and guess what, some people might write their name in Cyrillic...
> 
> My answer should be taken as advice. Meaning Cyrillic characters in fact I meant any characters in general other than Latin and special characters (allowed to use in file names in a your file system). This problem is common, so the best solution in my opinion is to stop using such characters in paths.
> 
> Best regards, Feniksovich from Russia :)

Ye, this is main reason why i changed names of folders. My friend had this problem too, in a retro game from 2007
